modules/ting_new_materials/ting_new_materials.module

```diff
@@ -150,25 +153,22 @@ function ting_new_materials_make_limit_query($limit = 0) {
  * @param int $num_per_page
  *   The page number.
  *
- * @return TingClientSearchResult
+ * @return TingSearchResultInterface
```

Comment:
This does not seem right. Obviously the return type of this method changes but I would expect that all the callers needs to be updated as well to use the methods provided by the new interface.